exercises/concept/black-jack/.docs/instructions.md

```diff
@@ -1,49 +1,85 @@
-In this exercise, you're going to implement some rules from [Blackjack][blackjack]
+# Instructions
 
-You have some rules to implement for judging the result of the game.
+In this exercise you are going to implement some rules of [Blackjack][blackjack],
+such as the way the game is played and scored.
 
-**Note** : In this exercise, _A_ means an ace, _J_ means jack, _Q_ means queen, and _K_ means king card.
+**Note** : In this exercise, _A_ means ace, _J_ means jack, _Q_ means queen, and _K_ means king cards.
 
-## 1. Calculate the number of card
+## 1. Calculate the value of a card
 
-Create the `number_of_card()` function with a parameter `card`. The value of _J, Q_ or _K_ is 10. If the `card` is _A_, then just return "ace".
+In Blackjack, it is up to each individual player if an ace is worth 1 or 11 points.
+Face cards are worth 10 points and any other card is worth its pip value.
+
+Define the `value_of_card` function with a parameter `card`.
+The value of _J_, _Q_ or _K_ is 10.
+Otherwise, return the pip value of a card.
+Ace can take multiple values so let's ignore _A_ for the time being.
 
 ```python
->>> number_of_card('K')
+>>> value_of_card('K')
 10
->>> number_of_card('A')
-ace
+>>> value_of_card('4')
+4
 ```
 
-## 2. Calculate the number of Ace
-
-Create the `number_of_ace()` function with a parameter `hand`.
+## 2. Calculate the value of ace
 
-1. `hand` : the sum of cards in hand with an ace.
+As mentioned before, ace can be worth either 1 or 11 points.
+At the same time, players try to get as close to 21 as possible, without going over 21.
 
-Ace is 1 or 11. You have to decide the value of ace without the sum of hand exceeding 21.
+Define the `value_of_ace` function with a parameter `hand_value`,
+which is the total hand value before getting an ace card.
+You have to decide how to score the upcoming ace card so that your hand won't exceed 21 if possible.
 
 ```python
->>> number_of_ace(19)
+>>> value_of_ace(19)
 1
->>> number_of_ace(7)
+>>> value_of_ace(7)
 11
 ```
 
-## 3. Judge Blackjack
+## 3. Determine Blackjack
+
+If the first two cards are an ace and a ten-card, giving a count of 21 in two cards, it is known as blackjack.
+
+Define the `is_blackjack` function with parameters `card_one` and `card_two`, which are a pair of cards.
+Determine if the hand is a blackjack.
```

Comment:
```suggestion
Determine if the two card hand is a blackjack.
```

Reply:
I'm going to use the "two-card hand" expression for the sake of consistency.
```suggestion
Determine if the two-card hand is a blackjack.
```